next-app/src/components/views/Introduce/Introduce.style.tsx

```diff
@@ -13,57 +14,146 @@ export const Wrapper = styled.div`
     padding: 40px 20px;
   `}
 `
+
 export const Container = styled.div`
   display: flex;
-  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+
+  max-width: 860px;
 `
 
-export const Contents = styled.div`
-  display: flex;
-  justify-content: center;
-  gap: 80px;
-  max-width: 786px;
+export const HeaderTitle = styled.h1`
+  display: inline-block;
+
+  color: var(--light-gray-1000, #2a2528);
```

Comment:
```suggestion
  color: var(--color-gray-1000, #2a2528);
```
css variable 이름이 잘못 적용되어서 컬러모드에 따른 색상 변경이 되지 않고 있네요 🥲 아마 피그마에 light모드 버전으로 디자인이 제시 되어 있어서 그런것 같은데, css 변수명은 컬러모드 이름을 떼고 `var(--color-gray-1000)` 다음과 같이 작성해주면 컬러모드에 따라서 자동으로 색상이 변경됩니다. 전체 변수명은 dark.css나 light.css 파일을 참고해주세요!